Library/Tracking/KSRAnalytics.swift

```diff
@@ -42,8 +42,10 @@ public final class KSRAnalytics {
     case messages // MessagesViewController
     case onboarding // CategorySelectionViewController, CuratedProjectsViewController
     case pledgeAddNewCard = "pledge_add_new_card"
+    case pledgeManager = "pledge_manager" // PPOView
```

Comment:
Nit: Maybe delete this unless you know you need it later? Also, out of curiosity, what part of PPO counts as a "pledgeManager" context? I would never have guessed based on the name that that'd be related to PPO.

Reply:
Will delete, this was for a backend event and got added by mistake.